AJAX_PATTERN_CONSISTENCY.md

```
@@ -0,0 +1,92 @@
+# AJAX Pattern Consistency Guide
+
+## Standard Pattern
+
+All `ajaxPost()` calls now follow a consistent pattern throughout the application:
+
+### Basic Pattern (No loader needed)
+```javascript
+ajaxPost(url, data)
+    .done(function(response) {
+        // Handle success
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)